docs/deploy/includes/prerequisites.md

```diff
@@ -2,5 +2,5 @@
 
 {: .no_toc }
 
-Users that require S3 access using [virtual host addressing](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) should [configure an S3 Gateway domain](../../setup/virtual-host-addressing.md).
+Users that require S3 access using [virtual host addressing](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html) should [configure an S3 Gateway domain](../../setup/virtual-host-addressing.html).
```

Comment:
not sure the '.md' is the problem here

Reply:
Since this page is included in another markdown, the link is not compiled by Jekyll - and the "md" stays in the final html.

Reply:
my bad - it is the fact that this file is included - the page is relative to the include folder? or maybe check that you can reference /setup/virtual-host-addressing.html